src/index.js

```diff
@@ -55,6 +78,13 @@ class CreateDynamoDBGlobalTables {
   }
 
   async addReplica(tableName, region) {
+    const tableHasBeenUpgraded = await this.tableIsAtVersionNumber(tableName);
```

Comment:
hmm, that would mean that we can't add a replica if the Global Table is already at the latest version? that doesn't seem right 🤔